src/docfx/restore/RestoreMap.cs

```diff
@@ -53,7 +53,7 @@ public string GetUrlRestorePath(string docsetPath, string path)
         {
             Debug.Assert(!string.IsNullOrEmpty(path));
 
-            if (!HrefUtility.IsAbsoluteHref(path))
+            if (!HrefUtility.IsHttpHref(path))
```

Comment:
nit: IsHttp?

Reply:
consistent with IsAbsoluteHref?

---
In reply to: [203931119](https://github.com/dotnet/docfx/pull/3105#discussion_r203931119) [](ancestors = 203931119)